org.eclipse.scout.rt.ui.html.test/src/main/java/org/eclipse/scout/rt/ui/html/AbstractRestProxyRequestHandlerTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.ui.html;
+
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import org.eclipse.scout.rt.rest.resource.ApiExposedFilter;
+import org.eclipse.scout.rt.server.commons.servlet.HttpProxy;
+import org.eclipse.scout.rt.server.commons.servlet.HttpProxyRequestOptions;
+import org.junit.Test;
+import org.mockito.internal.util.MockUtil;
+
+public abstract class AbstractRestProxyRequestHandlerTest {
```

Comment:
javadoc? like Abstract test for {@link AbstractRestProxyRequestHandler} instances